<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1465</width>
    <height>542</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>271</width>
      <height>199</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Meter:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="editMeter"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Title:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="editNoteLength"/>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="editKey"/>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="editType"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="editTranscriber"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>URL:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editTitle"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Note Length:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Key:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="editUrl"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Transcriber:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>601</width>
      <height>103</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Directory:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Filename:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="editFilename"/>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="checkAutosave">
       <property name="text">
        <string>Autosave</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QWidget" name="widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="editDirectory"/>
        </item>
        <item>
         <widget class="QPushButton" name="buttonDir">
          <property name="text">
           <string>Dir...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="buttonSave">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="buttonNew">
         <property name="text">
          <string>New</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>591</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>The ABC Notation itself:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="editABC"/>
     </item>
    </layout>
   </widget>
   <widget class="QWebView" name="webView">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>20</y>
      <width>541</width>
      <height>441</height>
     </rect>
    </property>
    <property name="url">
     <url>
      <string>about:blank</string>
     </url>
    </property>
   </widget>
   <zorder>gridLayoutWidget</zorder>
   <zorder>formLayoutWidget</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>label_11</zorder>
   <zorder>webView</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1465</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKitWidgets/QWebView</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>editDirectory</tabstop>
  <tabstop>buttonDir</tabstop>
  <tabstop>editFilename</tabstop>
  <tabstop>checkAutosave</tabstop>
  <tabstop>buttonSave</tabstop>
  <tabstop>buttonNew</tabstop>
  <tabstop>editTitle</tabstop>
  <tabstop>editTranscriber</tabstop>
  <tabstop>editUrl</tabstop>
  <tabstop>editType</tabstop>
  <tabstop>editMeter</tabstop>
  <tabstop>editNoteLength</tabstop>
  <tabstop>editKey</tabstop>
  <tabstop>editABC</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
